.github/workflows/e2e-compatibility-unreleased.yaml

```diff
@@ -0,0 +1,106 @@
+name: Compatibility E2E
+on:
+  workflow_dispatch:
+    inputs:
+      release-branch:
+        description: 'Select unreleased to run all release branches against each other'
+        required: true
+        type: choice
+        options:
+          - unreleased
```

Comment:
We can leave this as is, or remove it since we have one option, will leave it to you!